schema/deploy/warehouse/target.sql

```diff
@@ -0,0 +1,23 @@
+-- Deploy seattleflu/schema:warehouse/target to pg
+-- requires: warehouse/schema
+
+begin;
+
+set search_path to warehouse;
+
+create table target (
+    target_id integer primary key generated by default as identity,
+    identifier text not null unique,
+    control_status boolean not null
```

Comment:
Let's call this just `control`, in keeping with the naming conventions of our other boolean columns.